plugins/modules/clustering/pacemaker_resource.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Mathieu Bultel <mbultel@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: pacemaker_resource
+short_description: Manage a pacemaker resource
+extends_documentation_fragment: openstack
+version_added: "2.9"
+author: "Mathieu Bultel (matbu)"
+description:
+   - Manage a pacemaker resource from Ansible
+options:
+    state:
+      description:
+        - Indicate desired state of the cluster
+      choices: ['manage', 'unmanage', 'enable', 'disable', 'restart',
+                'show', 'delete', 'started', 'master', 'slave']
+      required: true
+    resource:
+      description:
+        - Specify which resource you want to handle
+      required: false
+      default: None
+    timeout:
+      description:
+        - Timeout when the module should considered that the action has failed
+      required: false
+      default: 300
+    check_mode:
+        description:
+          - Check only the status of the resource
+        required: false
+        default: false
+    wait_for_resource:
+        description:
+          - Wait for resource to get the required state, will failed if the
+            timeout is reach
```

Comment:
`I(...)` is how we use Italics to refer to another paramater
```suggestion
            I(timeout) seconds is reached.
```